src/Geometry.h

```diff
@@ -27,6 +27,19 @@
 class Cmfd;
 
 /**
+ * @enum ringifyType
+ * @brief The method to use when subdividing cells into rings
+ */
+enum ringifyType{
+
+  /** Max ring radius chosen such that it has the same area as the total geometry */
```

Comment:
The subdivide cells process is a recursive process that traverses the CSG tree, subdividing all cells filled with materials as it goes. When a cell located in some universe of some lattice cell is subdivided, the bounding box of the lattice cell is used to get the max radius. For pin cell problems that don't have any nesting, the bounding box of the root universe is used and this bounding box is computed implicitly via the Geometry::getWidthX()` method. I hope that makes sense!